Neuron/Sections/Transaction/SendTransactionViewController.swift

```diff
@@ -297,3 +306,49 @@ extension SendTransactionViewController: QRCodeViewControllerDelegate {
         addressTextField.text = codeResult
     }
 }
+
+// MARK: - Switch transaction token
+extension SendTransactionViewController: TransactionSwitchTokenViewControllerDelegate {
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        if indexPath.row == 0 {
+            if !enableSwitchToken {
+                return 0.0
+            } else {
+                return super.tableView(tableView, heightForRowAt: indexPath)
+            }
+        } else {
+            return super.tableView(tableView, heightForRowAt: indexPath)
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        if indexPath.row == 0 {
+            if enableSwitchToken {
+                cell.isHidden = false
+            } else {
+                cell.isHidden = true
+            }
+        } else {
+            cell.isHidden = false
+        }
+    }
+
+    func switchToken(switchToken: TransactionSwitchTokenViewController, didSwitchToToken token: TokenModel) {
+        self.token = token
+
+        observers.forEach { (observe) in
+            observe.invalidate()
+        }
+        observers.removeAll()
+
```

Comment:
This part is duplicated from `init`. Can we reuse them by extracting into a function?